sdk/eventgrid/Azure.ResourceManager.EventGrid/CHANGELOG.md

```diff
@@ -10,6 +10,21 @@
 
 ### Other Changes
 
+## 1.1.0-beta.2 (2023-06-30)
+
+### Features Added
+- Add support for Namespace
+- Add support for Namespace Topics.
+- Add support for Namespace Topic subscriptions.
```

Comment:
- Add support for Namespace Topic EventSubscriptions.

Reply:
fixed